.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
Instead of adding this new `.flake8` file, I'll suggest just adding this to the list of ignored flake8 things in https://github.com/ros-infrastructure/rosdoc2/blob/main/test/test_flake8.py

Reply:
Good point. I've removed this file and ignored the warning in the test script instead (https://github.com/ros-infrastructure/rosdoc2/pull/43/commits/d60fb06db604962bb19ec54450baf3c777836d8e).